lib/eco/incparser/annotation.py

```diff
@@ -43,6 +43,18 @@ class ToolTip(Hint):
     pass
 
 
+class Eval(Hint):
```

Comment:
Will these classes (Eval and Types) have some concrete data in the future, or will they only ever be placeholders?